include/linux/if_packet.h

```diff
@@ -13,4 +13,76 @@ struct sockaddr_ll {
 	unsigned char sll_addr[8];
 };
 
+/* Packet types */
```

Comment:
I'd prefer that we copy a more recent version of this file from the linux kernel, instead of whatever this is.  The license header changed in late 2017.